### DISCUSSION TITLE
خطا یا نصب نشدن بعضی از آیتم ها بعد از نصب پنل

### DISCUSSION THREAD

**Discussion by alekibod** [hiddify/Hiddify-Manager#3140] [Bug باگ: please use issue]
سلام و ممنون و خسته نباشی به بجه های هیدی فای واقعا ازتون ممنونم.
وقتی پنل نصب میکنم و نصب تموم میشه .
سرویس های که اکتیو رو نشون میده ولی اینا این خطا رو میده .
Failed to get unit file state for clash-server.service: No such file or directory
Failed to get unit file state for ss-faketls.service: No such file or directory
Failed to get unit file state for ss-v2ray.service: No such file or directory
چجوری دستی نصبشون کنم.
یا چیکار کنم که این مشکل بوجود نیاد.
ممنون و تشکر فراوان.

**Comment by liMilCo** [comment]
من در سرور اول مشکل نداشتم اما بعد از انتقال سرور به این مشکل بر خردم:
Failed to get unit file state for clash-server.service: No such file or directory

که برنامه های کلش و هیدیفای کلش آپدیت نمیشن ...
